Examples/ExactDiag/dissipative_ising1.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2018 The Simons Foundation, Inc. - All Rights Reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#    http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import netket as nk
+import numpy as np
+
+np.set_printoptions(linewidth=180)
+rg = nk.utils.RandomEngine(seed=1234)
+
+# 1D Lattice
+L = 2
+g = nk.graph.Hypercube(length=L, n_dim=1, pbc=False)
+
+# Hilbert space of spins on the graph
+hi = nk.hilbert.Spin(s=0.5, graph=g)
+
+
+# Defining the Ising hamiltonian (with sign problem here)
+# Using local operators
+sx = [[0, 1], [1, 0]]
+sy = [[0, -1j], [1j, 0]]
+sz = [[1, 0], [0, -1]]
+s0 = [[0, 0], [0, 0]]
+
+sigmam = [[0, 0], [1, 0]]
+
+ha = nk.operator.LocalOperator(hi)
+j_ops = []
+
+for i in range(L):
+    ha += nk.operator.LocalOperator(hi, sx, [i])
+    ha += nk.operator.LocalOperator(hi, np.kron(sz, sz), [i, (i + 1) % L])
+    j_ops.append(nk.operator.LocalOperator(hi, sigmam, [i]))
+
+
+# Create the lindbladian with no jump operators
+lind = nk.operator.LocalLindbladian(ha)
+
+# add the jump operators
+for j_op in j_ops:
+    lind.add_jump_op(j_op)
+
+
+rho = nk.exact.steadystate(lind)
```

Comment:
sorry again for these remarks, but they are kind of important for consistency: this should be exact.steady_state